incubator/honeydipper/templates/daemon-deployment.yaml

```
@@ -0,0 +1,88 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ include "honeydipper.fullname" . }}
+  labels:
+    app.kubernetes.io/name: {{ include "honeydipper.name" . }}
+    helm.sh/chart: {{ include "honeydipper.chart" . }}
+    app.kubernetes.io/instance: {{ .Release.Name }}
+    app.kubernetes.io/managed-by: {{ .Release.Service }}
+spec:
+  replicas: {{ .Values.daemon.replicaCount }}
+  selector:
+    matchLabels:
+      app.kubernetes.io/name: {{ include "honeydipper.name" . }}
+      app.kubernetes.io/instance: {{ .Release.Name }}
+  template:
+    metadata:
+      labels:
+        app.kubernetes.io/name: {{ include "honeydipper.name" . }}
+        app.kubernetes.io/instance: {{ .Release.Name }}
+    spec:
+      {{- with .Values.daemon.spec }}
+      {{ toYaml . | nindent 6 }}
+      {{- end }}
+      containers:
+        {{- if .Values.drivers.redis.local }}
+        - name: {{ .Chart.Name }}-redis
+          image: "redis:5"
+        {{- end }}
+        - name: {{ .Chart.Name }}
+          image: "{{ .Values.daemon.image.repository }}:{{ .Values.daemon.image.tag }}"
+          imagePullPolicy: {{ .Values.daemon.image.pullPolicy }}
+          {{- with .Values.daemon.args }}
+          args:
+            {{ toYaml . | nindent 12 }}
```

Fix this accordingly through out the chart.
```suggestion
            {{- toYaml . | nindent 12 }}
```

---

Done. Thanks!